### ISSUE TITLE
VDP Viewer ignores some background events

### ISSUE THREAD

**Issue by getCursorsExe** [TASVideos/BizHawk#2788]
### Summary
For unknown reasons, some background events to be ignored by the VDP Viewer, like this section of TiTAN Overdrive:

![image](https://user-images.githubusercontent.com/39802077/121216560-4c37e500-c881-11eb-8b67-ddaa0052ecd4.png)

Tried the same thing in Genesis Plus GX Debugger in BizHawk 2.4.2 (due to #2786), and it correctly displays them.
### Repro

1. Load TiTAN Overdrive ROM
2. Wait until you see the section in the image
3. Open VDP Viewer
4. Something appears to be wrong.
### Host env.
BizHawk 2.4.2 Win10
BizHawk 2.6.2 Win10

**Comment by nattthebear** [comment]
Removing regression tag:  The core it works in is a libretro one, and not the same as our native gpgx core.

**Comment by nattthebear** [comment]
On casual inspection this appears to be a raster effect, in which case the discrepancy is intentional:  Our graphics tools usually snapshot the visual state at a specific instant in time and display information about that instant, not the overall frame.

The thing we could add here is a scanline adjuster on when the VDP viewer snapshots, which would make this tool more in line with many of our other graphics tools.  If we did that, you'd be able to get the various rainbow colors depending on what scanline you stopped on, but the viewer would always be showing one solid rainbow of color.

**Comment by getCursorsExe** [comment]
Yeah, right now, it reads everything from the very first scanline, but Genesis Plus GX Debugger actually reads it from very bottom scanline.

**Comment by nattthebear** [comment]
That would mean it's not working correctly in Genesis Plus GX Debugger either.

**Comment by getCursorsExe** [comment]
Well, as far as I know, but that bottom frame happens to be the section where that raster effect is being processed, which is why it works correctly in Genesis Plus GX Debugger. In BizHawk's VDP Viewer, it has black background even though all frames are colorful.

**Comment by nattthebear** [comment]
Can you show a screenshot of what it looks like in Genesis Plus GX Debugger's VDP Viewer?

**Comment by getCursorsExe** [comment]
Yes.

**Comment by getCursorsExe** [comment]
![image](https://user-images.githubusercontent.com/39802077/121775238-a937e180-cb86-11eb-8ad4-2b603192a6ef.png)
